modules/caddyhttp/fileserver/staticfiles.go

```diff
@@ -47,6 +47,7 @@ type FileServer struct {
 	IndexNames    []string `json:"index_names,omitempty"`
 	Browse        *Browse  `json:"browse,omitempty"`
 	CanonicalURIs *bool    `json:"canonical_uris,omitempty"`
+	DirArchives   []string `json:"dir_archives,omitempty"`
```

Comment:
I think the "archive download" feature should be moved into the Browse configuration, since we require browse to be enabled for this anyway.

Reply:
And what about the caddy file?
Would be something like this?

```go
case "dir_archives":
	if fsrv.Browse == nil {
		fsrv.Browse = new(Browse)
	}
	selectedArchives := h.RemainingArgs()
	if err := validateArchiveSelection(selectedArchives); err != nil {
		return nil, err
	}
	fsrv.Browse.DirArchives = selectedArchives
```

Reply:
Sure, yeah! Something like that looks good.